var/spack/repos/builtin/packages/mpip/package.py

```diff
@@ -4,29 +4,121 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack import *
-import os
 
 
 class Mpip(AutotoolsPackage):
+
     """mpiP: Lightweight, Scalable MPI Profiling"""
-    homepage = "http://mpip.sourceforge.net/"
-    url      = "http://downloads.sourceforge.net/project/mpip/mpiP/mpiP-3.4.1/mpiP-3.4.1.tar.gz"
+    homepage = "https://software.llnl.gov/mpiP/"
+    url      = "https://github.com/LLNL/mpiP/releases/download/3.5/mpip-3.5.tgz"
     git      = "https://github.com/llnl/mpip.git"
+    maintainers = ['cchambreau']
 
     version('master', branch='master')
-    version("3.4.1", sha256="688bf37d73211e6a915f9fc59c358282a266d166c0a10af07a38a01a473296f0")
+    version('3.5',
+            sha256="e366843d53fa016fb03903e51c8aac901aa5155edabe64698a8d6fa618a03bbd")
+    version("3.4.1",
+            sha256="66a86dafde61546be80a130c46e4295f47fb764cf312ae62c70a6dc456a59dac")
+
+    conflicts('platform=darwin')
+
+    depends_on('python')
+    depends_on('python@:2', when='@:3.4.1')
+    depends_on('mpi')
+
+    #  Ideally would use libunwind, but provide backtrace and
+    #    setjmp functionality, if needed
+    #  depends_on('unwind')
+
+    variant('demangling', default=True,
+            description="Build with demangling support")
+
+    variant('setjmp', default=False,
+            description="Use setjmp to generate stack trace")
+
+    variant('mpi-io', default=True,
```

Comment:
I try to avoid variants with dashes in the name since dashes are also used to deactivate variants. Maybe an underscore would be better?